### ISSUE TITLE
Execute 'zr.painter.getRenderedCanvas({backgroundColor: '#fff'}).toDataURL()', which throw an error

### ISSUE THREAD

**Issue by bdwenxi** [ecomfe/zrender#701]
Execute '**zr.painter.getRenderedCanvas({backgroundColor: '#fff'}).toDataURL()**', which throw an error.

```
Uncaught TypeError: Cannot read property 'dpr' of undefined
```

**Comment by plainheart** [comment]
@bdwenxi Hi, thanks for your feedback! I think we've fixed it in #706.

**Comment by bdwenxi** [comment]
@plainheart thks~
